CalibTracker/Configuration/python/produceCalibrationTree_cfg.py

```diff
@@ -2,10 +2,15 @@
 
 process = cms.Process('CALIB')
 process.load('CalibTracker.Configuration.setupCalibrationTree_cff')
-process.load('Configuration.StandardSequences.Geometry_cff')
+process.load('Configuration.Geometry.GeometryExtended2015Reco_cff')
```

Comment:
@dimattia I suspect 
`process.load("Configuration.Geometry.GeometryRecoDB_cff")` 
Is enough for your use case. As discussed privately since you foresee to make further updates we can approve as is.